Content.Server/Fluids/EntitySystems/SpraySystem.cs

```diff
@@ -107,7 +105,8 @@ private void OnAfterInteract(EntityUid uid, SprayComponent component, AfterInter
 
             // impulse direction is defined in world-coordinates, not local coordinates
             var impulseDirection = Transform(vapor).WorldRotation.ToVec();
-            _vaporSystem.Start(vaporComponent, impulseDirection, component.SprayVelocity, target, component.SprayAliveTime);
+            var targetPlayerLocal = EntityCoordinates.FromMap(args.User, target, EntityManager);
+            _vaporSystem.Start(vaporComponent, impulseDirection, component.SprayVelocity, targetPlayerLocal, component.SprayAliveTime);
```

Comment:
Should this be changed to use MapCoordinates here?

Reply:
Probably.